linkerd/dns/name/src/name.rs

```diff
@@ -165,7 +165,7 @@ fn is_valid_reference_dns_id(hostname: untrusted::Input<'_>) -> bool {
 // https://tools.ietf.org/html/rfc5280#section-4.2.1.6:
 //
 //   When the subjectAltName extension contains a domain name system
-//   label, the domain name MUST be stored in the dNSName (an IA5String).
+//   label, the domain name MUST be stored in the DnsName (an IA5String).
```

Comment:
```suggestion
//   label, the domain name MUST be stored in the DNSName (an IA5String).
```
overeager find and replace for the `webpki` type, lol...